index.html

```diff
@@ -0,0 +1,308 @@
+<!doctype html>
+
+<html lang="en">
+	<head>
+		
+  		<title>My Portfolio</title>
+  		
+  		<link rel = "stylesheet" type = "text/css" href = "css/styles.css" />
+  		<link rel = "stylesheet" type = "text/css" href = "css/normalize.css" />
+
+	</head>
+	
+	<body>
+		<nav>
+			<div class="navSpacer">
```

Comment:
divs arent usually used in this manner - this could be done with margin-top on the element below